pkg/ds/daemon_set.go

```diff
@@ -161,8 +150,17 @@ func (ds *daemonSet) WatchDesires(
 					continue
 				}
 
-			case <-deletedCh:
-				ds.logger.NoFields().Infof("Received daemon set delete signal")
+			case deleteDS := <-deletedCh:
```

Comment:
Currently, if a daemon set is deleted, we will lose track of it, I plan to include a check in the daemon set farm during initialization that will unschedule nodes that are labelled with a daemon set uuid when their daemon set does not exist in the tree.